client/console/src/main/java/org/apache/syncope/client/console/panels/search/AbstractSearchPanel.java

```diff
@@ -179,18 +177,6 @@ protected void populate() {
             }
         };
 
-        anames = new LoadableDetachableModel<Map<String, PlainSchemaTO>>() {
```

Comment:
Removing `anames` from this parent class requires to put it in all its children, e.g. `AnyObjectSearchPanel`, `ConnObjectSearchPanel` (where you placed it) and `GroupSearchPanel` (where it is missing).